index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Tribute Page</title>
+  </head>
+  <body>
+    <main id="main">
+      <div id="title">
+        <h1>Dr. Norman Borlaug</h1>
+      </div>
+      <div id="img-div">
+        <img src="" alt="" id="image" />
+        <figcaption id="img-caption">
```

Comment:
i suggest you add an image link/path to your src to fix the broken img line in your css